src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  people: Person[],
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
```

Comment:
you should map people and find their mothers and fathers

Reply:
done, thank you!